bot.py

```diff
@@ -351,6 +353,31 @@ def __init__(self, *args, choices=None, opts_to_inv=None, **kwargs):
         )
         self.add_item(select_menu)
 
+class EmojiSyncView(discord.ui.View):
+    # Might be able to delete guild reference
+    def __init__(self, emoji, mod_type: str, old_name=None, *args, **kwargs):
+        super().__init__(timeout=None, *args, **kwargs)
+        self.emoji = emoji
+        self.old_name = old_name
+        
+        # Mod type will be a String of 'Add', 'Del', or 'Name' 
+        self.mod_type = mod_type
+
+    @discord.ui.button(label='Accept', style=discord.ButtonStyle.green)
+    async def accept_callback(self, button, interaction: discord.Interaction):
+        await interaction.response.send_message('Okay! I will sync this now', ephemeral=True)
+        
+        if self.mod_type == 'Add':
+            await sync_add(bot=bot, emoji=self.emoji)
+        elif self.mod_type == 'Del':
+            await sync_delete(bot=bot, emoji=self.emoji)
+        else:
+            await sync_name(bot=bot, old_name=self.old_name, new_emoji=self.emoji)
+
+    @discord.ui.button(label='Deny', style=discord.ButtonStyle.red)
```

Comment:
This button always seems to fail - no error in console.

Reply:
This button is probably "failing" because it is not responding to the interaction it is sent, which makes it just say "this interaction failed". In the Discord API, **all interactions must be responded to once and only once** or an error is shown to the user.

To prevent this, @josband, put an interaction response when `deny` button is clicked, like

```python
await interaction.response.send_message('This emoji will not be synced.', ephemeral=True)
```

Reply:
Thank you!